test/addIndex.js

```diff
@@ -68,7 +68,7 @@ describe('addIndex', function() {
     it('passes the index as an additional parameter', function() {
       eq(mapFilterIndexed(
         R.multiply,
-        R.gt(R.__, 13),
+        R.lte(13),
```

Comment:
First off, `R.gt(R.__, x)` is equivalent to `R.lt(x)`, not `R.lte(x)`.  It's an easy mistake to make, as that's how the converses of comparison operators work.  But this is not a converse.

But more importantly, I think if we don't introduce some other function such as `isGt` and simply crossreference them in the docs, then we show how to achieve this by using `gt`, either with `flip` or `partial`.